docs/spinner.md

```diff
@@ -2,19 +2,28 @@
 title: Spinner
 ---
 
+[cli-ux](https://github.com/oclif/cli-ux) provides a simple `cli.prompt()` function, for more complex input prompts, we recommend using the [listr](https://www.npmjs.com/package/listr) library.
```

Comment:
Looks like this got pasted from the prompt page and should be deleted or updated to reflect cli-ux.actions.

Reply:
Thanks for catching that. I'll plan to change `.prompt()` to `.action()` this evening unless I hear otherwise. The duplication across the pages was intentional, figuring that if it's valuable on the one page it's valuable on both

Reply:
Updated!